woocommerce-connect-client.php

```diff
@@ -406,7 +406,7 @@ public function attach_hooks() {
 			add_action( 'wc_connect_fetch_service_schemas', array( $schemas_store, 'fetch_service_schemas_from_connect_server' ) );
 			add_filter( 'woocommerce_hidden_order_itemmeta', array( $this, 'hide_wc_connect_package_meta_data' ) );
 			add_filter( 'is_protected_meta', array( $this, 'hide_wc_connect_order_meta_data' ), 10, 3 );
-			add_action( 'add_meta_boxes', array( $this, 'add_meta_boxes' ), 40 );
+			add_action( 'add_meta_boxes', array( $this, 'add_meta_boxes' ), 5 );
```

Comment:
If you leave this as it was but you change the `add_meta_box` `$priority` param in the `add_meta_boxes` from `'default'` to `'high'` (line 740), it should have the same effect. Up to you whatever you feel it's more robust. I'm ok if you :shipit: this.